lib/query-conversation/src/main/resources/solr/core/conversation/conf/schema.xml

```diff
@@ -28,7 +28,7 @@
     <field name="owner"                 type="string"         indexed="true"  stored="true"  multiValued="false"/>
     <field name="user_id"               type="string"         indexed="true"  stored="true"  multiValued="false"/>
     <field name="user_name"             type="string"         indexed="true"  stored="true"  multiValued="false"/>
-    <dynamicField name="env_*"          type="string"         indexed="true"  stored="true"  multiValued="false"/>
+    <dynamicField name="env_*"          type="string"         indexed="true"  stored="true"  multiValued="true"/>
```

Comment:
does that mean, we have to re-index everything when performing a release?

Reply:
Actually I am not sure how Solr reacts on such changes. Hoewever you will need to re-index anyways as their will be a migration script for Mongo that adds `ENV_SUPPORT_AREA` values for older messages.